lib/modules/manager/terragrunt/modules.ts

```diff
@@ -49,6 +54,15 @@ export function analyseTerragruntModule(
     dep.depName = 'github.com/' + dep.packageName;
     dep.currentValue = githubRefMatch.groups.tag;
     dep.datasource = GithubTagsDatasource.id;
+  } else if (gitlabRefMatch?.groups) {
+    dep.depType = 'repository';
+    dep.packageName = gitlabRefMatch.groups.project.replace(
+      regEx(/\.git$/),
+      '',
+    );
+    dep.depName = `${gitlabRefMatch.groups.host}/${dep.packageName}`;
```

Comment:
Are you sure you want the host name in the depName?

Reply:
Tried to keep the same approach like for github.com, I don't have any strong opinion here. What do you suggest?